apps/server/src/modules/files-storage/controller/api-test/files-storage-preview.api.spec.ts

```diff
@@ -0,0 +1,437 @@
+import { createMock, DeepMocked } from '@golevelup/ts-jest';
+import { EntityManager } from '@mikro-orm/mongodb';
+import { ExecutionContext, INestApplication, NotFoundException } from '@nestjs/common';
+import { Test, TestingModule } from '@nestjs/testing';
+import { ApiValidationError } from '@shared/common';
+import { EntityId, Permission } from '@shared/domain';
+import { AntivirusService } from '@shared/infra/antivirus/antivirus.service';
+import { cleanupCollections, mapUserToCurrentUser, roleFactory, schoolFactory, userFactory } from '@shared/testing';
+import { ICurrentUser } from '@src/modules/authentication';
+import { JwtAuthGuard } from '@src/modules/authentication/guard/jwt-auth.guard';
+import { FilesStorageTestModule } from '@src/modules/files-storage';
+import { FileRecordResponse } from '@src/modules/files-storage/controller/dto';
+import { Request } from 'express';
+import request from 'supertest';
+import { S3ClientAdapter } from '../../client/s3-client.adapter';
+import { ErrorType } from '../../error';
+import { TestHelper } from '../../helper/test-helper';
+import { PreviewOutputMimeTypes } from '../../interface/preview-output-mime-types.enum';
+
+class API {
+	app: INestApplication;
+
+	constructor(app: INestApplication) {
+		this.app = app;
+	}
+
+	async postUploadFile(routeName: string, query?: string | Record<string, unknown>) {
+		const response = await request(this.app.getHttpServer())
+			.post(routeName)
+			.attach('file', Buffer.from('abcd'), 'test.png')
+			.set('connection', 'keep-alive')
+			.set('content-type', 'multipart/form-data; boundary=----WebKitFormBoundaryiBMuOC0HyZ3YnA20')
+			.query(query || {});
+
+		return {
+			result: response.body as FileRecordResponse,
+			error: response.body as ApiValidationError,
+			status: response.status,
+		};
+	}
+
+	async postUploadFromUrl(routeName: string, data: Record<string, unknown>) {
```

Comment:
@bergatco Can it be removed because its not used?

Reply:
Removed!